### PR TITLE
java service discovery sample

### PR DESCRIPTION
**Pull request by Patrick0308**

### CODE REVIEW

File: registry/servicediscovery/zookeeper/java-server/src/main/java/org/apache/dubbo/User.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
+package org.apache.dubbo;
+
+import java.io.Serializable;
+import java.util.Date;
+
+/**
+ *@author Patrick Jiang
```
Comment (AlexStocks) [inline registry/servicediscovery/zookeeper/java-server/src/main/java/org/apache/dubbo/User.java:7]: pls delete the user info

File: registry/servicediscovery/zookeeper/java-server/src/main/java/org/apache/dubbo/User.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+ *@Description
+ *@date 2021/4/19 5:04 下午
```
Comment (AlexStocks) [inline registry/servicediscovery/zookeeper/java-server/src/main/java/org/apache/dubbo/User.java:9]: change it into english

File: registry/servicediscovery/zookeeper/java-server/src/main/java/org/apache/dubbo/UserProvider.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package org.apache.dubbo;
+
+import org.apache.dubbo.User;
+
+/**
+ *@author Patrick Jiang
+ *@Description
+ *@date 2021/4/19 4:57 下午
```
Comment (AlexStocks) [inline registry/servicediscovery/zookeeper/java-server/src/main/java/org/apache/dubbo/UserProvider.java:8]: the same as above. delete author. do not use chinese.

File: registry/servicediscovery/zookeeper/java-client/src/main/java/org/apache/dubbo/User.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
+package org.apache.dubbo;
```
Comment (zouyx) [inline registry/servicediscovery/zookeeper/java-client/src/main/java/org/apache/dubbo/User.java:1]: add lisence

File: registry/servicediscovery/zookeeper/java-client/src/main/java/org/apache/dubbo/UserProvider.java [hunk 5]
```diff
@@ -0,0 +1,13 @@
+package org.apache.dubbo;
```
Comment (zouyx) [inline registry/servicediscovery/zookeeper/java-client/src/main/java/org/apache/dubbo/UserProvider.java:1]: lisence

File: registry/servicediscovery/zookeeper/java-client/src/main/resources/spring/dubbo-consumer.xml [hunk 6]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```
Comment (zouyx) [inline registry/servicediscovery/zookeeper/java-client/src/main/resources/spring/dubbo-consumer.xml:2]: lisence moves to line 1

File: registry/servicediscovery/zookeeper/java-server/src/main/java/org/apache/dubbo/User.java [hunk 7]
```diff
@@ -0,0 +1,68 @@
+package org.apache.dubbo;
```
Comment (zouyx) [inline registry/servicediscovery/zookeeper/java-server/src/main/java/org/apache/dubbo/User.java:1]: lisence

File: registry/servicediscovery/zookeeper/java-server/src/main/java/org/apache/dubbo/UserProvider.java [hunk 8]
```diff
@@ -0,0 +1,13 @@
+package org.apache.dubbo;
```
Comment (zouyx) [inline registry/servicediscovery/zookeeper/java-server/src/main/java/org/apache/dubbo/UserProvider.java:1]: lisence

File: registry/servicediscovery/zookeeper/java-server/src/main/java/org/apache/dubbo/UserProviderImpl.java [hunk 9]
```diff
@@ -0,0 +1,21 @@
+package org.apache.dubbo;
```
Comment (zouyx) [inline registry/servicediscovery/zookeeper/java-server/src/main/java/org/apache/dubbo/UserProviderImpl.java:1]: lisence

File: registry/servicediscovery/zookeeper/java-server/src/main/resources/spring/dubbo-provider.xml [hunk 10]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```
Comment (zouyx) [inline registry/servicediscovery/zookeeper/java-server/src/main/resources/spring/dubbo-provider.xml:2]: lisence moves to line 1

**Review by zouyx — COMMENTED** [review]
make checking sucess

File: registry/servicediscovery/zookeeper/java-client/src/main/resources/spring/dubbo-consumer.xml (hunk 6 above)
Comment (Patrick0308) [reply registry/servicediscovery/zookeeper/java-client/src/main/resources/spring/dubbo-consumer.xml:2]: In java, it is ok.

**Pull request closed without merging**